global-news-app/src/assets/styles/shared.ts

```diff
@@ -9,4 +9,5 @@ export const sharedStyleDateSite = css`
   color: ${theme.colors.secondary.textColorHover};
 `;
 
+export const SelectWidth = 175;
 export const DATE_FORMAT = "EEEE MMM dd, yyyy";
```

Comment:
i thing the better place will be utils folder
you can create a consts file

or you can keep this const where you keep the format date function